### PR TITLE
Witches Strike - Inscribe Keyword, Tuning, Manifest text update

### PR DESCRIPTION
**Pull request by LyrenScalari**

### CODE REVIEW

File: src/main/java/thePackmaster/cardmodifiers/witchesstrikepack/InscribedMod.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import thePackmaster.orbs.PackmasterOrb;
+
+public class InscribedMod extends AbstractCardModifier {
+    private CardStrings uiStrings =  CardCrawlGame.languagePack.getCardStrings(SpireAnniversary5Mod.makeID("Inscribed"));
+    private boolean inherent = false;
+    private boolean Todesc = true;
+    public InscribedMod(boolean inherent,boolean todesc) {
+        Todesc = todesc;
+    }
+    public void onUpdate(AbstractCard card) {
```
Comment (erasels) [inline src/main/java/thePackmaster/cardmodifiers/witchesstrikepack/InscribedMod.java:24]: I don't think they should have a special glow.

File: src/main/java/thePackmaster/cards/WitchesStrike/CrescentSweep.java [hunk 2]
```diff
@@ -22,6 +29,26 @@ public CrescentSweep() {
...
+                public void update() {
+                    if (AbstractDungeon.player.hand.size() > 0) {
+                        ArrayList<AbstractCard> Uninscribed = new ArrayList<>();
+                        for (AbstractCard c : AbstractDungeon.player.hand.group){
+                            if (!CardModifierManager.hasModifier(c,"Inscribed")){
+                                Uninscribed.add(c);
+                            }
+                        }
+                        if (!Uninscribed.isEmpty()) {
+                            AbstractCard targetCard = Uninscribed.get(AbstractDungeon.miscRng.random(Uninscribed.size() - 1));
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/WitchesStrike/CrescentSweep.java:44]:
```suggestion
                            AbstractCard targetCard = Uninscribed.get(AbstractDungeon.cardRandomRng.random(Uninscribed.size() - 1));
```

File: src/main/java/thePackmaster/cards/WitchesStrike/HornetWithin.java [hunk 3]
```diff
@@ -30,7 +35,24 @@ public void use(AbstractPlayer p, AbstractMonster m) {
...
+            public void update() {
+                if (AbstractDungeon.player.hand.size() > 0) {
+                    ArrayList<AbstractCard> Uninscribed = new ArrayList<>();
+                    for (AbstractCard c : AbstractDungeon.player.drawPile.group){
+                        if (!CardModifierManager.hasModifier(c,"Inscribed")){
+                            Uninscribed.add(c);
+                        }
+                    }
+                    if (!Uninscribed.isEmpty()) {
+                        AbstractCard targetCard = Uninscribed.get(AbstractDungeon.miscRng.random(Uninscribed.size() - 1));
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/WitchesStrike/HornetWithin.java:49]:
```suggestion
                        AbstractCard targetCard = Uninscribed.get(AbstractDungeon.cardRandomRng.random(Uninscribed.size() - 1));
```

File: src/main/java/thePackmaster/cards/WitchesStrike/MysticFlourish.java [hunk 4]
```diff
@@ -17,16 +23,35 @@ public MysticFlourish() {
...
+                public void update() {
+                    if (AbstractDungeon.player.hand.size() > 0) {
+                        ArrayList<AbstractCard> Uninscribed = new ArrayList<>();
+                        for (AbstractCard c : AbstractDungeon.player.hand.group){
+                            if (!CardModifierManager.hasModifier(c,"Inscribed")){
+                                Uninscribed.add(c);
+                            }
+                        }
+                        if (!Uninscribed.isEmpty()) {
+                            AbstractCard targetCard = Uninscribed.get(AbstractDungeon.miscRng.random(Uninscribed.size() - 1));
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/WitchesStrike/MysticFlourish.java:43]:
```suggestion
                            AbstractCard targetCard = Uninscribed.get(AbstractDungeon.cardRandomRng.random(Uninscribed.size() - 1));
```

File: src/main/java/thePackmaster/cards/WitchesStrike/WitchTwist.java [hunk 5]
```diff
@@ -22,11 +29,31 @@ public WitchTwist() {
...
+                public void update() {
+                    if (AbstractDungeon.player.hand.size() > 0) {
+                        ArrayList<AbstractCard> Uninscribed = new ArrayList<>();
+                        for (AbstractCard c : AbstractDungeon.player.hand.group) {
+                            if (!CardModifierManager.hasModifier(c, "Inscribed")) {
+                                Uninscribed.add(c);
+                            }
+                        }
+                        if (!Uninscribed.isEmpty()) {
+                            AbstractCard targetCard = Uninscribed.get(AbstractDungeon.miscRng.random(Uninscribed.size() - 1));
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/WitchesStrike/WitchTwist.java:49]:
```suggestion
                            AbstractCard targetCard = Uninscribed.get(AbstractDungeon.cardRandomRng.random(Uninscribed.size() - 1));
```

File: src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json [hunk 6]
```diff
@@ -1,37 +1,38 @@
...
   },
   "${ModID}:FullMoonHalo": {
     "NAME": "Full Moon Halo",
-    "DESCRIPTION": "Gain !B! Block. NL Draw a card. NL Manifest ${ModID}:Full_Moon."
+    "DESCRIPTION": " Gain !B! Block. NL Manifest ${ModID}:Full_Moon."
   },
   "${ModID}:MoonlightBarrage": {
     "NAME": "Moonlight Barrage",
-    "DESCRIPTION": "Activate the passive effect of all your orbs. NL Deal !D! damage to a random enemy for each."
+    "DESCRIPTION": "Deal !D! damage to a random enemy for each."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json:23]: what?

File: src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json [hunk 7]
```diff
@@ -1,37 +1,38 @@
...
   },
   "${ModID}:HornetWithin": {
     "NAME": "Hornet Within",
-    "DESCRIPTION": "${ModID}:Infest. NL Gain !B! block. NL When this ${ModID}:infests, activate the passive effect of your next orb."
+    "DESCRIPTION": "${ModID}:Infest. NL Gain !B! block. NL When this ${ModID}:infests, ${ModID}:Inscribe a random card in your draw pile."
   },
   "${ModID}:WitchTwist": {
     "NAME": "Witch Twist",
-    "DESCRIPTION": "Activate the passive effect of your next orb !M! times. NL Gain !anniv5:m2! Focus."
+    "DESCRIPTION": "${ModID}:Inscribe !M! random card in your hand. NL Gain !anniv5:m2! Focus."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json:35]:
```suggestion
    "DESCRIPTION": "${ModID}:Inscribe !M! random card in your hand. NL Gain !${ModID}:m2! Focus."
```

**Review by erasels — CHANGES_REQUESTED** [review]

File: src/main/java/thePackmaster/cardmodifiers/witchesstrikepack/InscribedMod.java (hunk 1 above)
Comment (LyrenScalari) [reply src/main/java/thePackmaster/cardmodifiers/witchesstrikepack/InscribedMod.java:24]: noted. this was just something copied from the mod the mechanic came from will remove it.

File: src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json (hunk 6 above)
Comment (LyrenScalari) [reply src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json:23]: need to word it like Barrage. forgot the Moonlight Barrage relied on having it's effect spelled out. and doesn't work with Inscribed out the box

**Review by erasels — APPROVED** [review]

**Pull request merged**